crates/vhost-device-gpio/src/backend.rs

```diff
@@ -240,8 +241,15 @@ fn start_backend(args: GpioArgs) -> Result<()> {
     Ok(())
 }
 
+#[cfg(feature = "mock_gpio")]
+const DEFAULT_LOG: &str = "info";
+
+#[cfg(not(feature = "mock_gpio"))]
+const DEFAULT_LOG: &str = "warn";
```

Comment:
Would just using `info` as standard level for all features and nerfing unwanted log levels from the _real_ chip down to debug allow us to avoid changing the default level by feature?

Reply:
Done